integration_tests/suite/helpers/base.py

```diff
@@ -77,48 +94,78 @@ def restart_postgres(cls):
         until.true(database.is_up, timeout=5, message='Postgres did not come back up')
 
 
-class AutoConfiguredDirdTestCase(DBRunningTestCase):
+class BaseDirdIntegrationTest(DBRunningTestCase):
 
+    wait_strategy = RestApiOkWaitStrategy()
     config_factory = new_null_config
+    service = 'dird'
 
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.mock_auth_client = MockAuthClient(
-            '127.0.0.1', cls.service_port(9497, 'auth')
+        cls.host = '127.0.0.1'
+        cls.port = cls.service_port(9489, 'dird')
+        if cls.asset not in ('documentation', 'short-self-stop'):
```

Comment:
We should not hardcode asset name here. If you want special configuration for these asset, you just need to create antoher class with BaseDirdIntegration as inheritance